lib/stubApi.ts

```diff
@@ -11,7 +11,7 @@ export type { MemberPublic };
 interface MemberPublicDupe {
   name?: string;
   companySize?: string;
-  emailAbbr?: string[];
+  emailAbbr?: string;

```

Comment:
What was the reason for this being an array to begin with?

Reply:
used to pull the return value of [useEmailCloaker()](https://github.com/hawaiians/hawaiiansintech/blob/26d6427771cb808a0ad225f2402eaa09dfd28e26/helpers.js#L65) which basically gets the beginning and end char of the user's email and passes that as an array of strings, but now we save the email as a full string